tests/integration-tests/tests/performance_tests/test_starccm/test_starccm/starccm.install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#Bash script used to execute the installation of starCCM+ on the headnode
```

Comment:
Super minor: strictly speaking this is not true: the script installs StarCCM+ in a shared folder that is shared with all the cluster nodes, not only in the head node.